docs/components/modeler/bpmn/send-tasks/send-tasks.md

```diff
@@ -27,13 +27,14 @@ a job worker.
 
 ## Defining a task
 
-A send task must define a [job type](/components/modeler/bpmn/service-tasks/service-tasks.md#task-definition); it specifies the type of job that workers should subscribe to (e.g. `kafka`).
+A send task must define a [job type](/components/modeler/bpmn/service-tasks/service-tasks.md#task-definition) the same way as a service task does. It specifies
+the type of job that workers should subscribe to (e.g. `kafka` or `mail`).
 
 Use [task headers](/components/modeler/bpmn/service-tasks/service-tasks.md#task-headers) to pass static parameters to the job
 worker (e.g. the name of the topic to publish the message to).
 
-Define [variable mappings](/components/modeler/bpmn/service-tasks/service-tasks.md#variable-mappings) to transform the
-variables passed to the job worker, or to customize how the variables of the job merge in the process instance.
+Define [variable mappings](/components/concepts/variables.md#inputoutput-variable-mappings) to transform the
+variables passed to the job worker, or to customize how the variables of the job merge
```

Comment:
Thanks for your comment here. I think it is worth discussing this change and our opinions about this. 
I agree that the send task works like a service task and should therefore linked to that one. 
From a user point of view who wants to really understand what variable mappings are I do not think linking the service task topic is sufficient. I would really like to see a bit more details there. 

Even though the page I referenced is about how to define variable mappings it let's you understand the concept quite well.  

Do you have any opinion on enriching the service tasks variable mapping section with some general information?

Reply:
👍 I agree that we can improve it. Feel free to add additional general information. 

I was thinking a bit about this and perhaps we can link to both the concepts and the element definition.

For send-task's job type you changed the text to:
```
A send task must define a [job type](/components/modeler/bpmn/service-tasks/service-tasks.md#task-definition) 
the same way as a service task does.
```

If we would link to both the concept and the 'how to define', it could look like:
```
A send task must define a [job type](components/concepts/job-workers/)
the [same way as a service task does](/components/modeler/bpmn/service-tasks/service-tasks.md#task-definition).
```

We could also apply this to variable mappings
```
Define [variable mappings](/components/concepts/variables.md#inputoutput-variable-mappings) 
the [same way as a service task does](/components/modeler/bpmn/service-tasks/service-tasks.md#variable-mappings) 
to transform the variables passed to the job worker, or to customize how the variables of the job merge. 
```

WDYT?

Reply:
That is a great idea. 👍 
Since this change is going to affect some more files I am thinking about creating a whole new PR for this. 
So how about merging this one for now and I will drag you into a bigger one which focuses on variable-mapping only?